### DISCUSSION TITLE
Wyze 2FA API key for HomeAssistant Version

### DISCUSSION THREAD

**Discussion by cbc24** [mrlt8/docker-wyze-bridge#1074] [Q&A]
I see that I can use an API key for Wyze 2FA.  I've got the key, but I am unclear on where (e.g., which file) I would add these to the configuration.  Or does this not work with the Home Assistant version?

**Comment by mrlt8** [comment]
You'll need to toggle the show unused options in the add-on's config in home assistant.

**Reply by cbc24** [reply]
Perfect!  Thanks!
